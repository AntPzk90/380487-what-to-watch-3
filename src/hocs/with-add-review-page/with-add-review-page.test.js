import React from 'react';
import renderer from 'react-test-renderer';
import withAddReviewPage from './with-add-review-page.jsx';


it(`SnapshotTest withAddReviewPage`, () => {

  const tree = renderer
  .create(
      <withAddReviewPage
        onFormSubmit={() => {}}
        isBlocked={false}
        isShowMassege = {false}
        isBlockedForm = {false}
        isError = {false}
        onInputComment = {() => {}}
        onChangeRating = {() => {}}
        rating = {0}
        comment = {``}
      />
  )
  .toJSON();

  expect(tree).toMatchSnapshot();
});
