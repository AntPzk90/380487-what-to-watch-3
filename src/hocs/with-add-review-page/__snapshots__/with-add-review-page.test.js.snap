// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotTest withAddReviewPage 1`] = `
<withAddReviewPage
  comment=""
  isBlocked={false}
  isBlockedForm={false}
  isError={false}
  isShowMassege={false}
  onChangeRating={[Function]}
  onFormSubmit={[Function]}
  onInputComment={[Function]}
  rating={0}
/>
`;
